src/frontend/src/binder/expr/function.rs

```diff
@@ -170,6 +170,19 @@ impl Binder {
             }
         }
 
+        // datatype cast function
+        // only functions required by the existing PostgreSQL tool are implemented
+        #[allow(clippy::single_match)] // remove it when more cast functions needed
```

Comment:
FYI `expect` instead of `allow` would make "remove it when more cast functions needed" easier.